source/developers-guide/customer-streams-extension/index.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: default
+title: Customer - Search and Streams
+github_link: developers-guide/customer-streams-extension/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Customer - Search & Streams
+menu_order: 500
+---
+
+This document describes how to extend the customer stream and customer search part in shopware. In most cases, a plugin wants to provide more conditions or data for the search. This document describes, by way of a simple example, how this is possible.
```

Comment:
Alternativer Vorschlag: This article describes how to extend the customer stream and customer search module in Shopware. In most cases, the purpose of a plugin is to provide more conditions or to add data to the search. You find simple examples below to explain how this is achieved.